test/integration/index-test.js

```diff
@@ -250,6 +253,63 @@ describe(function() {
         assertNoUnstaged(status);
       });
 
+      it('can update a custom blueprint for an ember app project', async function() {
+        let originalSpawn = installAndGenerateBlueprint.spawn;
+        let originalResolvePackage = cliUpdateCommandModule.resolvePackage;
+        let originalIsDefaultAddonBlueprint = getStartAndEndModule.isDefaultAddonBlueprint;
+        let fakeAddonName = 'some-addon-with-blueprint';
+
+        sinon.stub(getStartAndEndModule, 'isDefaultAddonBlueprint').callsFake((blueprint) => {
```

Comment:
I'm having trouble understanding why all this mocking is needed in this test.

Reply:
Maybe using a local addon would help

```
      "name": "ember-cli-update-git-blueprint-test",
      "location": "../local-blueprint",
```

Reply:
I updated the test to remove all the mocking. Let me know what you think.

Reply:
oh nice, that's way cleaner.